src/style.css

```
@@ -0,0 +1,43 @@
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  height: 16px;
+  margin: auto;
+  width: 100px;
+}
+
+.stars__star {
+  width: 16px;
+  background-image: url(./images/Star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star:nth-child(-n+0) {
```

```suggestion
.stars--0 .stars__star:nth-child(-n+0),
.stars--1 .stars__star:nth-child(-n+1),
... {
```
Consider using commas to group selectors